man/deepsea-monitor.1

```diff
@@ -0,0 +1,66 @@
+.TH deepsea-monitor 1 "DeepSea CLI man page"
+.SH NAME
+deepsea-monitor - Starts the stage execution monitor.
```

Comment:
"Starts the DeepSea stage execution progress monitor" ? (also see next comment, below)